testsuite/tests/mgc/reconciliation/test_gw_doesnt_exist.py

```diff
@@ -0,0 +1,40 @@
+"""Tests that DNSPolicy/TLSPolicy is rejected if the Gateway does not exist at all"""
+
+import pytest
+
+from testsuite.gateway import CustomReference
+from testsuite.policy.dns_policy import DNSPolicy
+from testsuite.tests.mgc.reconciliation import tls_policy
+
+pytestmark = [pytest.mark.mgc]
+
+
+@pytest.fixture(scope="module", autouse=True)
+def commit():
+    """We want to manage Policies on our own here"""
+    return
+
+
+@pytest.mark.parametrize(
+    "create_cr", [pytest.param(DNSPolicy.create_instance, id="DNSPolicy"), pytest.param(tls_policy, id="TLSPolicy")]
+)
+@pytest.mark.issue("https://github.com/Kuadrant/multicluster-gateway-controller/issues/361")
+def test_dns_no_gw(request, create_cr, hub_openshift, blame, module_label):
+    """Tests that DNSPolicy is rejected if the Gateway does not exist at all"""
```

Comment:
Looks like you parametrizing both policies but in comments/names/types only DNSPolicy mentioned